README.md

```diff
@@ -16,12 +16,106 @@ make env
 
 ## Development Server
 To run the development server, create a `local_settings.py` file as a sibling of `settings.py`. Add a `SECRET_KEY` value
-with some random stuff (e.g `SECRET_KEY = 'sjfio3u903RaggleFraggle'`).
+with some random stuff (e.g `SECRET_KEY = 'sjfio3u903RaggleFraggle'` and see "Environment Variables" section).
 
 The Django local development can be run via:
 
 ```bash
 make watch
 ```
 
+Another means to run local is the manage.py from within the wellregistry path:
 
+```bash
+python -m manage runserver
+```
+Note that environment variable must be configured to connect to a postgres database.
+
+To run tests locally:
+
+```bash
+python -m manage test
+```
+
+
+## Environment Variables
+
+    This project has a private companion config project where environment variable
+    files for each deployment tier are maintained. The values in the config files are
+    applied to the docker container for each teir at deployment. 
+    The following sub sections will describe the various variables used.
+
+General
+
+    DEBUG: boolean - true for debug level logging
+
+Cloud
+
+    These environment variables are required for the cloud deployment.
+    They are not used for local development.
+
+        SECRET_KEY: AWS secret key
```

Comment:
It's somewhat inaccurate to say that SECRET_KEY and ALLOWED_HOSTs is for cloud deployment. They are for deployment anywhere.

Reply:
I do not need them for my local development.

Reply:
My point is that they are not cloud specific.

Reply:
I still do not follow you. They are AWS config settings. How are they not cloud specific? If we deployed on-prem we would not require them, right?

Reply:
No ... They are both used for EROS applications like NEMI.

Reply:
odd. ok.

Reply:
changed "cloud" to "tier"

Reply:
The `SECRET_KEY` is always required. The only reason local development doesn't fail without a developer specifying one is because one is randomly generated when the app is run locally.

Reply:
Then it must not be the AWS secret key that this is referencing?

Reply:
I am not being combative here. I am operating on what I thought the values looked like since I did not require them for local deployment.
How about this? These values were in the conf before I added the database entries: ALLOWED_HOSTS, CIDR_RANGES, SECRET_KEY
Just tell me what they are or what we want as a description and I will gladly update them. I took the best shot at what I thought they were.

Reply:
The Django documentation contains the definitions for SECRET_KEY and ALLOWED_HOSTS:  https://docs.djangoproject.com/en/3.0/ref/settings/. The CIDR_RANGES is used to set ALLOWED_CIDR_NETS which is documented here: https://github.com/mozmeao/django-allow-cidr

Reply:
I found some Django settings docs that look to describe ALLOWED_HOSTS and SECRET_KEY.
https://docs.djangoproject.com/en/3.0/ref/settings/#allowed-hosts
https://docs.djangoproject.com/en/3.0/ref/settings/#std:setting-SECRET_KEY
While I did not find CIDR_RANGES there, are these two correct. I could paraphrase or link to these pages. Do I have CIDR_RANGES correctly described?

Reply:
I stated above where to look for CIDR_RANGES. It is used to set ALLOWED_CIDR_NETS defined in https://github.com/mozmeao/django-allow-cidr . But I think the description is fine.

Reply:
I think  a brief description is better than linking to the docs which are tied to a version of django.

Reply:
Thanks Mary. When I posted my reply stating "I found some Django settings..." above, the github thread was not up to date. I had gone back to the same browser window without refreshing and missed your reply.

Reply:
Ok, updated once more to describe the Django nature of these settings.

Reply:
https://github.com/mozmeao/django-allow-cidr  readme says that its middleware entry should be first in the list. It is currently last. I could move it. Should I move it?